### ISSUE TITLE
[Issue] BTKUILib doesn't work in 2023r171ex3 due to CoHTML changes.

### ISSUE THREAD

**Issue by DjShinter** [BTK-Development/BTKUILib#6]
This is an example error log of my mod that uses BTKUILib "AnimationLoader"
```
[18:29:48.177] [AnimationLoader] System.MissingFieldException: Field not found: cohtml.CohtmlView ABI_RC.Core.InteractionSystem.CVR_MenuManager.quickMenu Due to: Could not find field in class
  at BTKUILib.UIObjects.Page.set_MenuTitle (System.String value) [0x0000f] in <33803db9c10342a8b88bae99c5d3d621>:0
  at AnimationLoader.AL.SetupUI () [0x00016] in <e19b186aad4b4b9f88145ee937ef33ca>:0
  at AnimationLoader.AL.SetupIcons () [0x0001e] in <e19b186aad4b4b9f88145ee937ef33ca>:0
  at AnimationLoader.AL.OnInitializeMelon () [0x00000] in <e19b186aad4b4b9f88145ee937ef33ca>:0
  at MelonLoader.MelonBase.LoaderInitialized () [0x00000] in D:\a\MelonLoader\MelonLoader\MelonLoader\Melons\MelonBase.cs:435
  ```
  >18:29:48.102] [BTKUILib] BTKUILib is starting up!
[18:29:48.161] [BTKUILib] Applied patches!
[18:29:48.162] [BTKUILib] Checking if BTKUI is updated...
[18:29:48.170] [BTKUILib] BTKUI is up to date!

Looks like it loads fine but when you initialize a menu it will start to error out.

Using v1.1.0 BTKUILib

**Comment by ddakebono** [comment]
Are you attempting to use BTKUILib with CVR Experimental? That is entirely unsupported and will very likely have critical issues. Can you post your whole MelonLoader log?

**Comment by ddakebono** [comment]
Wait I'm blind, you are. I'm not awake enough to be looking at GitHub issues lol

This is a known issue, most mods are not yet ready to support Unity 2022 or the experimentals, support will not be coming for quite a while still.

**Comment by ddakebono** [comment]
Fixed as of version 1.2.0 now that r171 has released
